framework/tests/all-systems/GetClassTest.java

```diff
@@ -6,11 +8,29 @@ void context() {
         Integer i = 4;
         i.getClass();
         Class<?> a = i.getClass();
-        // Type agruments don't match
-        @SuppressWarnings("fenum:assignment.type.incompatible")
+        // Type arguments don't match
+        @SuppressWarnings({
+            "fenum:assignment.type.incompatible",
+            "nullness:assignment.type.incompatible"
```

Comment:
Shouldn't this work for nullness? adaptGetClassReturnTypeToReceiver changes the upper bound of getClass to be non-null, so `Class<? extends Object>` should be a supertype of `Class<? extends Integer>`.
Same issues with other tests.